libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -59,6 +64,31 @@ void VotePacketHandler::process(const PacketData &packet_data, const std::shared
     } else {  // standard vote -> vote_round >= current_pbft_round
       if (vote_mgr_->voteInVerifiedMap(vote)) {
         LOG(log_dg_) << "Vote " << vote_hash.abridged() << " already inserted in verified queue";
+        // We don't need to process this vote because it is already in verified map
+        continue;
+      }
+
+      if (vote->getRound() > pbft_mgr_->getPbftRound() + kVoteAcceptingRounds) {
+        LOG(log_nf_) << "Skip. Round too big " << pbft_mgr_->getPbftRound() << " " << vote->getRound();
+        // Do not request round sync too often here
+        if (std::chrono::system_clock::now() - round_sync_request_time_ > std::chrono::seconds(10)) {
```

Comment:
those 10s could be another const variable

Reply:
and is this ok, to mark this vote at this time as seen? line `if (!seen_votes_.insert(vote_hash)) {` , because if this vote comes again, it won't be processed